src/containers/Review/assignment/ReviewSectionRow.tsx

```diff
@@ -88,18 +88,10 @@ const ReviewSectionRow: React.FC<ReviewSectionRowProps> = ({
     <div key={`section_${sectionCode}_assignment_${assignmentId}`}>
       {canRenderRow && (
         <Grid stackable columns={4} verticalAlign="middle">
-          <Grid.Column className="custom-assignment-grid-column">
```

Comment:
I think I added this in an earlier PR.  I hadn't realized that the components in the grid.col where actually returning grid.columns themselves. So removed them.